docs/source/docker/advanced/pretagging.rst

```diff
@@ -0,0 +1,136 @@
+
+The results of pretagging are visualized in the report. We report both, the 
+object distribution before and after the selection process. 
+
+The following image shows an example of such a histogram for the input data
+before filtering.
+
+.. figure:: ../resources/pretagging_histogram_example.png
+    :align: center
+    :alt: some alt text
+
+    Histogram plot of the pretagging model for the input data (full dataset).
+    The plot shows the distribution of the various detected classes. 
+    Further it shows the average number of objects per image.
+
+For every docker run with pretaggin enabled we also dump all model predictions
+into a json file with the following format:
+
+.. code-block:: json
+
+    [
+        {
+            "filename": "0000000095.png",
+            "boxes": [
+                [
+                    869.5512084960938,
+                    153.78746032714844,
+                    885.997314453125,
+                    197.8896026611328
+                ],
+                [
+                    231.59332275390625,
+                    175.98333740234375,
+                    291.9855651855469,
+                    202.55943298339844
+                ]
+            ],
+            "labels": [
+                "person",
+                "car"
+            ],
+            "scores": [
+                0.9845203757286072,
+                0.9323102831840515
+            ]
+        },
+        ...
+    ]
+
+
+Usage
+---------------
+
+Pretagging can be activated by passing the following argument to your docker
+run command: `pretagging=True`
+
+- `pretagging=True` enables the use of the pretagging model
+- `pretagging_debug=True` add a few images to the report for debugging showing
+  showing the image with the bounding box predictions. 
+
+The final docker run command to enable pretagging as well as pretagging_debug
+should look like this:
+
+.. code-block:: console
+
+   docker run --gpus all --rm -it \
+      -v OTHER_INPUT_DIR:/home/input_dir:ro \
+      -v SHARED_DIR:/home/shared_dir \
+      -v OUTPUT_DIR:/home/output_dir \
+      lightly/sampling:latest \
+      token=MYAWESOMETOKEN \
+      pretagging=True \
+      pretagging_debug=True
+
+The following shows an example of how the debugging images in the report look like:
+
+.. figure:: ../resources/pretagging_debug_example.png
+    :align: center
+    :alt: some alt text
+
+    The plot shows the detected bounding boxes from the pretagging overlayed
+    on the image. Use the debug feature to figure out whether the pretagging 
+    mechanism works properly on your dataset.
+
+
+Pretagging for Selection
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can also use pretagging to guide the data selection process. This can beh
+helpful if you for example only care about images where there is at least one
+person.
+
+To create such a pretagging selection mechanism you need to create a config file.
+
+For the example of selecting only images with >1 person we can create
```

Comment:
I need to double check the implementation but pretty sure it's >=

Reply:
yes, you're right. It's >=1